services/ui-src/README.md

```diff
@@ -1,56 +1,42 @@
-1. Run the app-api service locally. See [APP API README](../app-api/README.md) for more information.
-1. Install all the UI dependencies by running `npm install`.
-1. Run the application by running `npm run start` and open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+Be sure to follow the setup instructions for the OneMAC API, found [here](https://github.com/CMSgov/onemac/blob/develop/services/app-api/README.md). Also ensure that the build directory (`public`) has a file named `env-config.js` populated with a list of environment variables. These variables are used in the connection between the UI and API.
 
-### Logging in to the Application
+## Run the UI locally
 
-The application provides a set of test users for the DEV environment that you can use for all feature branches and the develop branch, including when running the application locally.
+1. Run the `app-api` service locally.
+2. Install all the UI dependencies by running `npm install` in this directory.
+3. Run the application by running `npm run start` from this directory.
 
-**Note that these users are not present in the master or production branches.**
+Running the start command will open [http://localhost:3000](http://localhost:3000) where UI changes can be viewed in real-time.
 
-The following are the available static test users:
+## Local testing accounts
 
-```
-statesubmitterpending@cms.hhs.local
-statesubmitteractive@cms.hhs.local
-statesubmitterdenied@cms.hhs.local
-statesubmitterrevoked@cms.hhs.local
-statesubmitterunregistered@cms.hhs.local
-statesystemadminpending@cms.hhs.local
-statesystemadminactive@cms.hhs.local
-statesystemadmindenied@cms.hhs.local
-statesystemadminrevoked@cms.hhs.local
-statesystemadminunregistered@cms.hhs.local
-cmsroleapproverpending@cms.hhs.local
-cmsroleapproveractive@cms.hhs.local
-cmsroleapproverdenied@cms.hhs.local
-cmsroleapproverrevoked@cms.hhs.local
-cmsroleapproverunregistered@cms.hhs.local
-```
+### Static test users
 
-For every build, a dynamic list of developer emails is also added to the test user list, so developers can receive the generated emails sent from the application. Developers need to setup their correct email in the Git configuration and have already committed changes to the repo. To see the list of emails added run the following command in the Git repository.
+We have a suite of [OneMAC test users](https://qmacbis.atlassian.net/wiki/spaces/MACPRO/pages/2446852097/oneMAC+Test+Users) for the DEV environment that you can use for all feature branches and the `develop` branch, including when running the application locally. If you are unable to access the Confluence page, see a teammate for help!
 
-`git log --pretty=format:'%ae' | grep -v github.com | sort -u`
+> **NOTE:** These users are not available for use in the `master` or `production` branch environments.
```

Comment:
Actually, test users *are* available in VAL (master) now...

Reply:
Ooo, good to know! I'll fix it up 👍🏼